Convert `osu!standard` > `osu!` everywhere (#5255)
ppy/osu-wiki: wiki/Beatmap_Editor/Compose/en.md

**Your construction toolkit, made for DIY (do it yourself).**

> **Please note that in the osu!'s default notes, hitsounds works differently in osu!mania.**

Reviewer: This still doesn't quite work

```suggestion
**Please note that in osu!, default note hitsounds work differently than in osu!mania.**
```

Author: I had not seen him, thank you

> **Please note that in the osu!'s mode default notes hitsounds works differently in osu!mania.**

Reviewer 2: This isn't good as an english sentence. The original *and* this change are wrong.
